pkgs/development/web/nodejs/v18.nix

```
@@ -23,8 +23,8 @@ let
 in
 buildNodejs {
   inherit enableNpm;
-  version = "18.20.4";
-  sha256 = "sha256-p2x+oblq62ljoViAYmDICUtiRNZKaWUp0CBUe5qVyio=";
+  version = "18.20.5";
+  sha256 = "76037b9bad0ab9396349282dbfcec1b872ff7bd8c8d698853bebd982940858bf";
```

By convention, it should use the `SRI` format for the hash
```
$ nix-hash --to-sri --type sha256 "76037b9bad0ab9396349282dbfcec1b872ff7bd8c8d698853bebd982940858bf"
sha256-dgN7m60KuTljSSgtv87BuHL/e9jI1piFO+vZgpQIWL8=
```

```suggestion
  version = "18.20.5";
  sha256 = "sha256-dgN7m60KuTljSSgtv87BuHL/e9jI1piFO+vZgpQIWL8=";
```

---

Nah, we should use the hash signed by Node.js releaser, which is the one I'm using here